test/test_rospkg_catkin_packages.py

```diff
@@ -54,7 +54,17 @@ def test_find_packages():
 
 
 def test_get_manifest():
-    search_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'catkin_package_tests'))
     manager = rospkg.rospack.ManifestManager(rospkg.common.MANIFEST_FILE, ros_paths=[search_path])
     manif = manager.get_manifest("foo")
     assert(manif.type == "package")
+
+
+def test_licenses():
+    rospack = rospkg.rospack.RosPack(ros_paths=[search_path]) 
+    licenses_list = ["BSD", "LGPL"]
+    licenses_str = ", ".join(licenses_list)
+    manif = rospack.get_manifest("foo")
+    print("DEBUG: manif.licenses={}".format(manif.licenses))
+    assert(manif.license == licenses_str)
```

Comment:
Please avoid trivial single use variables and use `", ".join(licenses_list)` here directly.

Reply:
This comment has not been addressed yet.